### DISCUSSION TITLE
Multiple Single-Port NICs or One Multi-Port NIC

### DISCUSSION THREAD

**Discussion by Gramarin** [Security-Onion-Solutions/securityonion#12179] [Unsupported Versions] [answered]
So the documentation states that to get Security Onion up and running, it requires 2 NICs. I'm talking about the most basic, minimalist installs here. I was wondering, is it possible to use a single multi-port NIC instead of multiple sing-port NICs?

**Comment by Acewiza** [comment]
It won't matter unless bus bandwidth moving gigs of traffic is an issue.  Sounds like maybe not, in ur case.

**Reply by Gramarin** [reply]
Nope, small business network with small amounts of traffic.

**Comment by reyesj2** [comment, marked as answer]
Yes, you can use Security Onion with 1 NIC that has multiple ports.
